app/app.jsx

```diff
@@ -2,14 +2,27 @@ import React from 'react';
 import './scss/style.scss';
 import 'normalize.css';
 
+/* import components */
+import Main from './containers/main.jsx';
+import Nav from './containers/nav.jsx';
+
 class App extends React.Component {
+
    render() {
+
+     const store = this.props.store;
```

Comment:
@des-des... i'm quite confused. Although not at all surprised that I'm doing lots of things wrong.
I've managed to refactor some stuff as in the tutorial above. However, now I have no idea how to access the state at all... can you help? Would it be ok if I push up my (broken) code to another branch so that you can take a look?